plugin.video.popcorntimes/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.popcorntimes" version="2.0" name="Popcorntimes.tv" provider-name="V10lator">
+<requires>
+<import addon="xbmc.python" version="3.0.0"/>
+<import addon="script.module.requests" version="2.12.4"/>
+<import addon="script.module.beautifulsoup4" version="4.3.2"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en_GB">Video plugin for popcorntimes.tv</summary>
+  <description lang="en_GB">Video plugin for popcorntimes.tv</description>
+  <disclaimer lang="en_GB">Video plugin for popcorntimes.tv</disclaimer>
+  <summary lang="de_DE">Video plugin für popcorntimes.tv</summary>
+  <description lang="de_DE">Video plugin für popcorntimes.tv</description>
+  <disclaimer lang="de_DE">Video plugin für popcorntimes.tv</disclaimer>
+  <license>GNU GENERAL PUBLIC LICENSE. Version 2</license>
```

Comment:
please use a spdx compliant identifier for the license: GPL-2.0-only or GPL-2.0-or-later

Reply:
Thanks for the heads up, I changed it.